### PR TITLE
Add contribution of new object types

### LINKED ISSUE #58: README: add notes on collaboration model
Say, I create a JSON schema for the ACID object type, will it as such be accepted into main?

There are 1000+ different object types(?), and each object type has an owner inside SAP(?)

### PR DESCRIPTION
**Pull request by schneidermic0**
Object type owners are invited to create pull requests.

Closes #58.

### CODE REVIEW

**Comment by larshp** [comment]
Say, I create the type for the ACID object type, will it as such be accepted into main?

**Comment by schneidermic0** [comment]
> Say, I create the type for the ACID object type, will it as such be accepted into main?

As mentioned in #58. I think the object owners shall to be involved. We are in contact with further teams for contributions, but they have to plan it. I did not reach out to colleagues who are responsible for ACID, yet. ;)

**Comment by larshp** [comment]
ok, ACID was just an example, not that its particularly needed

**Comment by schneidermic0** [comment]
> ok, ACID was just an example, not that its particularly needed

I considered it as example

**Review by albertmink — APPROVED** [review]
I like the content.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Feel free to raise issues to ask questions or report bugs.
 ## Contributing
 
 Comments and suggestions for improvements are most welcome.
+
+You are an owner of an ABAP object type? We will be more than happy if you contribute your object type to this project. Just open a new pull request.
```
Comment (xtough) [inline README.md:45]:
```suggestion
You feel responsible for an ABAP object type? We will be more than happy if you contribute your object type to this project. Just open a new pull request. 
```

**Review by xtough — CHANGES_REQUESTED** [review]
I would rephrase it slightly to open it up to external contributors. Owners are clearly limitted to SAP employees.

File: README.md (hunk 1 above)
Comment (schneidermic0) [reply README.md:45]: Yes, I see your point, but I think we have to involve the object owners. At least they have to review it. Should we add this somewhere?

Comment (xtough) [reply README.md:45]: Sure, but getting PRs and reviewing them are 2 different issues. Here, we primarily want to invite others, including object owners and external contributors, to collaborate. It goes without saying that any PR will be reviewed by us as maintainers and other experts as required. But I don't see a need to stress this point here.

**Comment by xtough** [comment]
see comment
